rds-postgres/admin-login/variables.tf

```diff
@@ -31,6 +31,11 @@ variable "read_principals" {
   default     = null
 }
 
+variable "replica_identifier" {
+  description = "Identifier of the database replica"
+  type        = string
```

Comment:
I think you'll want a default of `null` here for logins that don't use a replica.